build/source/engine/snowLiqFlx.f90

```diff
@@ -128,7 +128,7 @@ subroutine snowLiqFlx(&
  end if
 
  ! check the meltwater exponent is >=1
- if (mw_exp<1_i4b) then; err=20; message=trim(message)//'meltwater exponent < 1'; return; end if ! mw_exp is an i4b integer
+ if (mw_exp<1._rkind) then; err=20; message=trim(message)//'meltwater exponent < 1'; return; end if
```

Comment:
Another `i4b` to `rkind` change

Reply:
This revision was made for similar reasoning as above. Due to Fortran's mixed mode rules, `mw_exp<1_i4b` and `mw_exp<1._rkind`  will have identical values. However, the original version (`mw_exp<1._rkind`) correctly implies that `mw_exp` is of type `rkind`, which may be clearer for developers.